src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/matcher/FilterableResolver.java

```diff
@@ -101,6 +103,16 @@ public FilterableResolver() {
   }
 
   protected List<DBFilterBase> resolve(CommandContext commandContext, T command) {
+    List<DBFilterBase> filter = matchRules.apply(commandContext, command);
+    if (filter.size() > docLimits.maxObjectProperties()) {
```

Comment:
Should we add a separate limit instead of using existing one for this purpose?

If we use existing one, its Javadocs need to be updated to indicate this usage too.